goserver/server/server_test.go

```diff
@@ -76,3 +70,24 @@ func TestLaunchHandlerCheckISSAuthorization(t *testing.T) {
 		})
 	}
 }
+
+func TestHandleLaunch(t *testing.T) {
+	fhirURL := "https://authorized.fhir.com"
+	sm := session.NewInMemorySessionManager()
+	s := &Server{authorizedFHIRURL: fhirURL, sm: sm}
+	rr := httptest.NewRecorder()
+	req := httptest.NewRequest("GET", "/?iss="+fhirURL, nil)
+	s.handleLaunch(httptest.NewRecorder(), req)
+	if status := rr.Code; status != http.StatusOK {
+		t.Fatalf("server.handleLaunch returned wrong status code, got %v, want %v",
```

Comment:
I think Errorf would be more appropriate here, since the test can output more useful error info if it keeps running. Same fort the Fatalfs below.

Reply:
Done.